toqito/channels/depolarizing.py

```diff
@@ -88,7 +86,11 @@ def depolarizing(dim: int, param_p: float = 0) -> np.ndarray:
 
     """
     # Compute the Choi matrix of the depolarizing channel.
+    result = np.zeros((dim**2, dim**2), dtype=np.float64)
```

Comment:
I cannot directly suggest a change but can you please elaborate on what `param_p` is in the docstring? I am talking about this line: `:param param_p: Default 0.`